### ISSUE TITLE
Wrong Prometheus pod label

### ISSUE THREAD

**Issue by vielktus** [coreos/prometheus-operator#1297]
**What did you do?**
helm install -n kube-prometheus

**What did you expect to see?**
```
$ kubectl describe pod prometheus-kube-prometheus-prometheus-0
Labels:         app=prometheus
                controller-revision-hash=prometheus-kube-prometheus-prometheus-c6d9ddb8
                prometheus=kube-prometheus-prometheus <----- This should be "kube-prometheus" only
```

**What did you see instead? Under which circumstances?**
prometheus=kube-prometheus-prometheus 

I checked the service and found out that it didn't have any endpoints.
The "kube-prometheus-prometheus" service will chose the pod with label:
```
 Selector:          app=prometheus,prometheus=kube-prometheus
```
But the pod label was wrongly created like I mentioned above.
Did I miss configure anything or the Operator has some issues ?

My current value for "prometheusLabelValue" is "" (blank). I just leave it.

**Comment by verwilst** [comment]
I have the same issue. Kubespray-installed Kubernetes 1.10.2,

Ran following commands:

```
```

**Comment by verwilst** [comment]
Also tried to set a  prometheusLabelValue during helm install, but it doesn't seem to affect the pod label, which remains "kube-prometheus-prometheus".

```
---
prometheus:
   prometheusLabelValue: "prometheus-verwilst"
```

**Comment by vielktus** [comment]
@richerve I might need your help 👍

**Comment by gianrubio** [comment]
@vielktus you can solve this issue removing the selector `prometehus` from the prometheus service. Leave only the selector `app: prometheus`.

I'm gonna raise a PR to fix that.

**Comment by brancz** [comment]
can this be closed as #1302 is merged?

**Comment by verwilst** [comment]
Installed 0.0.36:

kubectl describe svc kube-prometheus-prometheus -n monitoring
<snip>
Labels:            app=prometheus
                   chart=prometheus-0.0.36
                   heritage=Tiller
                   prometheus=kube-prometheus
                   release=kube-prometheus
</snip>
TargetPort:        9090/TCP
Endpoints:         <none>


kubectl describe pod prometheus-kube-prometheus-prometheus-0 -n monitoring
<snip>
Labels:         app=prometheus
                controller-revision-hash=prometheus-kube-prometheus-prometheus-5484998ff9
                prometheus=kube-prometheus-prometheus
                statefulset.kubernetes.io/pod-name=prometheus-kube-prometheus-prometheus-0
</snip>

Looks like it's still an issue?

**Comment by gianrubio** [comment]
Sorry for any inconvenience, the prometheus object was fixed but not the service, I hope #1305 will fix that.

**Comment by vielktus** [comment]
Thank you @gianrubio.

**Comment by gianrubio** [comment]
Fixed by #1305
